src/stories/2-frontend-engine/frontend-engine.stories.tsx

```diff
@@ -220,8 +221,25 @@ ValidateOnBlur.args = {
 	},
 };
 
-export const OnChange: Story<IFrontendEngineProps> = (args: IFrontendEngineProps) => <FrontendEngine {...args} />;
+export const ValidateOnTouched = Template.bind({});
```

Comment:
this can be the default story now, and the old default will be a `ValidateOnSubmit` story